fsspec/implementations/tests/test_dbfs.py

```diff
@@ -23,6 +23,10 @@
 import os
 from urllib.parse import urlparse
 
+import numpy
+import pyarrow as pa
+import pyarrow.dataset as ds
```

Comment:
As discussed, this will fail for some. They should be
```
pa = pytest.importorskip("pyarrow")
```
inside the functions that require them.